src/GeneratedMapper/Helpers/IndentWriterExtensions.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.CodeDom.Compiler;
+
+namespace GeneratedMapper.Helpers
```

Comment:
Could you move these extensions to the GeneratedMapper.Extensions namespace (and folder)? I've grouped all extensions in that folder.